DistantWorlds2.ModLoader.ModManager/ModManager.cs

```diff
@@ -613,6 +616,31 @@ private static void HideMessageDialog(object o, DWEventArgs dwEventArgs)
 
   public ConcurrentDictionary<string, object> SharedVariables { get; } = new();
 
+  private byte[] CalculateGameDataChecksum()
+  {
+    var checksumHasher = new Crc64(); //TODO: Switch back to XxHash64 once https://github.com/dotnet/runtime/issues/69184 is resolved and released.
```

Comment:
```suggestion
    var checksumHasher = ModInfo.GetHasher();
```

N.B.

Reply:
Might move ModInfo.Hasher / GetHasher to ModManager or somewhere higher in the load order?

Reply:
GetHasher is a static factory method. It will give you a new object I think.